force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -206,8 +228,8 @@ public with sharing class VariablesDataTypesOperators {
         // Add parentheses in the below expression to change the result.
         // The result of the expression as it is right now is 43. 
         // You should add parentheses so that the result of the expression becomes 8.
-        Integer answer = 48 - 15 + 5 * 2;
-        return null; // Replace null with the variable you used to store the result
+        Integer answer = 48 - ((15 + 5) * 2);
```

Comment:
The outer parentheses aren't necessary here because multiplication will happen before subtraction automatically, following the order of operations